torchci/test/drciUtils.test.ts

```diff
@@ -52,6 +52,7 @@ describe("Test various utils used by Dr.CI", () => {
       await hasSimilarFailures(
         job,
         emptyBaseCommitDate,
+        [],
```

Comment:
sorry is there any testing for this? am i just looking in the wrong place

Reply:
Let me add a test for this, moving `mergeCommits` is actually a boon here because I can now easily mock it